src/common/config_opts.h

```diff
@@ -1136,7 +1136,8 @@ OPTION(bluestore_cache_type, OPT_STR, "2q")   // lru, 2q
 OPTION(bluestore_2q_cache_kin_ratio, OPT_DOUBLE, .5)    // kin page slot size / max page slot size
 OPTION(bluestore_2q_cache_kout_ratio, OPT_DOUBLE, .5)   // number of kout page slot / total number of page slot
 OPTION(bluestore_cache_size, OPT_U64, 1024*1024*1024)
-OPTION(bluestore_cache_meta_ratio, OPT_DOUBLE, .9)
+OPTION(bluestore_cache_meta_ratio, OPT_DOUBLE, .5)
```

Comment:
shouldn't we increase bluestore_cache_size with that default rocksdb_cache_size value and specify default ratios in a way that resulting disposition is similar to the one we have by default before. I.e. 128M to rocksdb, 900Mb to metadata and 100Mb to data cache. Just to not mess the feature and param tuning...